index.bs

```diff
@@ -2981,9 +2981,69 @@ Wikipedia also describes some <a href="https://en.wikipedia.org/wiki/Monkey_patc
 <h4 id="cant-avoid-monkey-patching">If you need to monkey patch</h4>
 Sometimes monkey patching is unavoidable (e.g., early in the design phase of a new specification). If you find yourself needing to monkey patch an existing specification, make sure you:
 
- 1. File a bug, or create a fix, for the existing specification. This will help you understand what the problem is and how to fix it, and maybe even alleviate you from even creating the monkey patch in the first place.
- 1. In your spec, clearly mark the monkey patch as a temporary workaround (e.g., using a big red box) and make it clear to readers that you intend to add your patch to the existing specification. Make sure you link to the existing specification and the bug or fix that you filed.
- 1. Work together with the authors of the existing specification on a solution. If the existing specification is no longer maintained... Congratulations! you are now the editor of that existing specification. If necessary, add suitable extension points to the existing specification to better handle your changes.
+ 1. In your spec, clearly mark the monkey patch as a proposed change to another specification that is only temporarily in this specification, using language like:
+
+    <div class="example" id="example-mark-monkeypatch">
+
+    ```markdown
+    # Modify HTTP fetch
+
+    <p class="issue">Merge this into [[Fetch]] once it has enough consensus.</p>
+    ```
+
+    </div>
+
+ 1. Identify the place you want to change using a quote from its text or by linking to a defined term if you're replacing that term's whole definition. Include the current step numbers if you're changing something that's numbered, but remember that numbers aren't sufficient by themselves because they change when other steps are added or removed. If you're changing several places within a definition, it can be helpful to paste the entire definition into a `<blockquote>` and use `<ins>` and `<del>` to describe what's changing.
+
+    <div class="example" id="example-monkeypatch-quote-text-before-step">
+
+    Before step 4.2 of [=HTTP fetch=], «If request's redirect mode is "follow", then set request's service-workers mode to "none".», insert the following steps:
```

Comment:
Use normal or curly-quotes instead of French (?)/Infra-list quotes?

Reply:
French, to avoid ambiguity with the quotes around the redirect modes. But they're not that ambiguous, so I can just use normal quotes. Done.